CMakeLists.txt

```diff
@@ -111,6 +111,7 @@ option(ALSOFT_UTILS "Build utility programs"  ON)
 option(ALSOFT_NO_CONFIG_UTIL "Disable building the alsoft-config utility" OFF)
 
 option(ALSOFT_EXAMPLES  "Build example programs"  ON)
+option(ALSOFT_TESTS "Build test programs"  ON)
```

Comment:
I don't think this should be on by default, since it tries to download extra stuff and I don't think most builds would need or want it.

Reply:
updated!